src/app/components/math-editor/index.tsx

```diff
@@ -130,10 +130,6 @@ export default function MathEditor(props: Props) {
               e.preventDefault()
               e.stopPropagation()
               close('after')
-            } else if (e.key === 'Enter' && containerRef.current) {
```

Comment:
IIRC this `&& containerRef.current` check needs to be somewhere between onEnter and spawnMathEditorOnNewLine 🤔